### PR TITLE
feat: Changes for 1.0.0

### PR DESCRIPTION
**Pull request by satlead**
## What
Removes /opt and other overrides for manifest making them compile time build configuration
Upgrades the developer run scripts with right build flags
Upgrades version numbers

## Why

First step to make Ripple production ready.

## How

Using Rust configuration flags

## Test
Execute ./ripple run <ip_address> 

## Checklist

- [ ] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

File: core/main/src/bootstrap/manifest/device.rs [hunk 1]
```diff
@@ -35,10 +35,12 @@ impl LoadDeviceManifestStep {
...
 
 fn try_manifest_files() -> Result<DeviceManifest, RippleError> {
-    let dm_arr: DeviceManifestLoader = if cfg!(test) {
+    let dm_arr: DeviceManifestLoader = if cfg!(any(feature = "local_dev", feature = "pre_prod")) {
+        vec![load_from_env, load_from_home, load_from_opt, load_from_etc]
+    } else if cfg!(test) {
         vec![load_from_env]
     } else {
-        vec![load_from_env, load_from_home, load_from_opt, load_from_etc]
+        vec![load_from_etc]
```
Comment (pahearn73) [inline core/main/src/bootstrap/manifest/device.rs:43]: Will this prevent the ability to debug prod releases using overridden manifests if necessary?

File: core/main/src/bootstrap/manifest/extn.rs [hunk 2]
```diff
@@ -30,10 +30,12 @@ impl LoadExtnManifestStep {
 type ExtnManifestLoader = Vec<fn() -> Result<(String, ExtnManifest), RippleError>>;
 
 fn try_manifest_files() -> Result<ExtnManifest, RippleError> {
-    let dm_arr: ExtnManifestLoader = if cfg!(test) {
+    let dm_arr: ExtnManifestLoader = if cfg!(any(feature = "local_dev", feature = "pre_prod")) {
```
Comment (pahearn73) [inline core/main/src/bootstrap/manifest/extn.rs:33]: What will be the expected differences between `local_dev` and `pre_prod`?

File: core/main/src/bootstrap/manifest/device.rs (hunk 1 above)
Comment (satlead) [reply core/main/src/bootstrap/manifest/device.rs:43]: Debug releases probably can be enabled with Rust log of debug or trace for triaging. Having this Prod builds opens up concerns with security this reduces the attack radius on a prod builds

File: core/main/src/bootstrap/manifest/extn.rs (hunk 2 above)
Comment (satlead) [reply core/main/src/bootstrap/manifest/extn.rs:33]: Probably one for the future where we can configure local dev shell scripts and pre-prod environments differently.
`pre_prod` can be used by ci cd pipelines to generate builds.

We can configure other items for local_dev alone to make it customizable felt its a good tangent

**Review by pahearn73 — APPROVED** [review]

**Review by kevinshahfws — APPROVED** [review]

**Pull request merged**